src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationConfigurationManager.cs

```diff
@@ -0,0 +1,44 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Tokens;
+
+namespace AspNet.Security.OAuth.SuperOffice
+{
+    /// <summary>
+    /// Represents the base class for loading configuration and obtaining keys for use with Sign in with SuperOffice.
+    /// </summary>
+    public abstract class SuperOfficeAuthenticationConfigurationManager
```

Comment:
This doesn't appear to be used for anything by the provider.

Reply:
It can be added if and when needed. We don't want extra unused code to maintain for a "maybe in the future" feature you may or may not implement.